implementation-reports/wordpress.md

```diff
@@ -0,0 +1,203 @@
+
+Implementation Home Page URLs:
+https://wordpress.org/plugins/webmention/
+https://wordpress.org/plugins/semantic-linkbacks/
+
+Source Code repo URL(s):
+https://github.com/pfefferle/wordpress-webmention/
+https://github.com/acegiak/Semantic-Linkbacks
+https://github.com/pfefferle/wordpress-semantic-linkbacks
+
+* [x] 100% open source implementation
+
+Programming Language(s): PHP
+
+Developer(s):
+* [Matthias Pfefferle](http://notizblog.org/)
+* [Ashton McAllan](https://acegiak.net/)
+* [David Shanske](https://david.shanske.com/)
+* [and more...](https://github.com/pfefferle/wordpress-webmention/graphs/contributors)
+
+Implementation Classes: Sender and Receiver
+
+
+## Sending
+
+### Discovery Tests (3.1.1)
+
+MUST
+
+* [x] [Discovery Test #1](https://webmention.rocks/test/1)
+* [x] [Discovery Test #2](https://webmention.rocks/test/2)
+* [x] [Discovery Test #3](https://webmention.rocks/test/3)
+* [x] [Discovery Test #4](https://webmention.rocks/test/4)
+* [x] [Discovery Test #5](https://webmention.rocks/test/5)
+* [x] [Discovery Test #6](https://webmention.rocks/test/6)
+* [x] [Discovery Test #7](https://webmention.rocks/test/7)
+* [x] [Discovery Test #8](https://webmention.rocks/test/8)
+* [x] [Discovery Test #9](https://webmention.rocks/test/9)
+* [x] [Discovery Test #10](https://webmention.rocks/test/10)
+* [x] [Discovery Test #11](https://webmention.rocks/test/11)
+* [x] [Discovery Test #12](https://webmention.rocks/test/12)
+* [x] [Discovery Test #13](https://webmention.rocks/test/13)
+* [x] [Discovery Test #14](https://webmention.rocks/test/14)
+* [x] [Discovery Test #15](https://webmention.rocks/test/15)
+* [x] [Discovery Test #16](https://webmention.rocks/test/16)
+* [x] [Discovery Test #17](https://webmention.rocks/test/17)
+* [x] [Discovery Test #18](https://webmention.rocks/test/18)
+* [x] [Discovery Test #19](https://webmention.rocks/test/19)
+* [x] [Discovery Test #20](https://webmention.rocks/test/20)
+* [x] [Discovery Test #21](https://webmention.rocks/test/21)
+
+
+### Sending Tests (3.1.2)
+
+MUST
+
+* [x] Accepts HTTP 200 response as a success
+* [x] Accepts HTTP 201 response as a success
+* [x] Accepts HTTP 202 response as a success
+
+
+### Update Tests (3.1.3)
+
+SHOULD
+
+* [x] [Update Test #1](https://webmention.rocks/update/1)
+* [ ] [Update Test #2](https://webmention.rocks/update/2)
+
+#### Implementation Notes
+
+Doesn't pass update test #2 yet because it doesn't send webmentions to removed links. [Tracked in issue #76.](https://github.com/pfefferle/wordpress-webmention/issues/76#issuecomment-225306427)
+
+
+### Delete Tests (3.1.4)
+
+SHOULD
+
+* [ ] [Delete Test #1](https://webmention.rocks/delete/1)
+
+#### Implementation Notes
+
+WordPress returns HTTP 404 for deleted posts, not 410.
+
+
+### Security Considerations (4)
+
+* [ ] The sender avoids sending a Webmention to a loopback address (SHOULD)
```

Comment:
Well it's in the security considerations section. It's more meant to be a warning for production systems.